prow/cmd/deck/main.go

```diff
@@ -116,6 +121,36 @@ func handleData(ja *JobAgent) http.HandlerFunc {
 
 type logClient interface {
 	GetJobLog(job, id string) ([]byte, error)
+	GetJobLogStream(job, id string, options map[string]string) (io.ReadCloser, error)
```

Comment:
we should

1) Document that options are passed straight through to the Kubernetes [read log](https://kubernetes.io/docs/api-reference/v1.6/#read-log) API.
2) Decide if that makes sense. Should we instead just pass follow=true? Does passing these parameters through allow better frontends?